### ISSUE TITLE
ImportError: cannot import name calc_lwork

### ISSUE THREAD

**Issue by shjoshi** [statsmodels/statsmodels#2184]
This is for statsmodels 0.6.1
I installed it using conda
This is the conda log:

    package                    |            build
     ---------------------------|-----------------
     six-1.9.0                  |           py27_0          17 KB
     pytz-2014.9                |           py27_0         175 KB
     numpy-1.9.1                |           py27_0         7.7 MB
     dateutil-2.1               |           py27_2         227 KB
     scipy-0.15.0               |       np19py27_0        37.0 MB
     patsy-0.3.0                |       np19py27_0         312 KB
     pandas-0.15.2              |       np19py27_0         9.9 MB
     statsmodels-0.6.1          |       np19py27_0         5.0 MB

The following NEW packages will be INSTALLED:

     dateutil:    2.1-py27_2
     numpy:       1.9.1-py27_0
     pandas:      0.15.2-np19py27_0
     patsy:       0.3.0-np19py27_0
     pytz:        2014.9-py27_0
     scipy:       0.15.0-np19py27_0
     six:         1.9.0-py27_0
     statsmodels: 0.6.1-np19py27_0

Then when I simply do:

    import statsmodels.api as sm

I get the following error:

  File "/XX/lib/python2.7/site-packages/statsmodels/api.py", line 15, in <module>
    from .tsa import api as tsa
  File "/XX/lib/python2.7/site-packages/statsmodels/tsa/api.py", line 5, in <module>
    from .vector_ar.var_model import VAR
  File "/XX/lib/python2.7/site-packages/statsmodels/tsa/vector_ar/var_model.py", line 23, in <module>
    from statsmodels.tools.linalg import logdet_symm
  File "/XX/lib/python2.7/site-packages/statsmodels/tools/linalg.py", line 23, in <module>
    from scipy.linalg import calc_lwork
ImportError: cannot import name calc_lwork

**Comment by hadiasghari** [comment]
I have the exact same issue. I also tried:

    conda uninstall statsmodels
    pip install statsmodels

The error remains the same when importanting statsmodels.api. Perhaps some dependency issue?

**Comment by josef-pkt** [comment]
It's and incompatibility with scipy 0.15 that we didn't catch in time
see #2159 and #2160 

fix is in master, but most likely there won't be an statsmodels 0.6.2 release for this, @jseabold  ?

**Comment by hadiasghari** [comment]
As a workaround, this works. ** After ** installing the latest statsmodels via conda, as stated in the original post, then run this:

    conda install scipy=0.14


Now the import works. Got to play around with it to see if anything else breaks or not.

**Comment by jseabold** [comment]
I can do a 0.6.2. But I'm away from my windows box indefinitely, so that will take some doing.

**Comment by stonebig** [comment]
Hi,

I fall also on the same issue when using ggplot, which uses statsmodels.

==> ggplot 0.6.5 cannot use statsmodels0.6.1 + scipy 0.15.0

==> falling back to scipy 0.14.1 for the moment.

Typical failure example:
````
from ggplot import *
import pandas as pd
meat_lng = pd.melt(meat[['date', 'beef', 'pork', 'broilers']], id_vars='date')
ggplot(aes(x='date', y='value', colour='variable'), data=meat_lng) + \
    geom_point() + \
    stat_smooth(color='red')
````

**Comment by matthew-brett** [comment]
Maybe it would be worth adding a statsmodels check here:

* https://github.com/MacPython/scipy-stack-osx-testing
* https://travis-ci.org/MacPython/scipy-stack-osx-testing

I'm happy to give y'all access if you want to do that.  And / or run against pre-built scipy (etc) dev versions as one of the test matrix entries on travis-ci - that is also easy to do using wheels, maybe it can be done with conda too.

**Comment by josef-pkt** [comment]
@matthew-brett    Thanks, I was just thinking about how we can test statsmodels master against scipy master in an automatic way, instead of relying on that someone is running it.

Is this only Mac or are there also pre-build scipy for Ubuntu for Travis-CI?

I'll look at it.

**Comment by matthew-brett** [comment]
The scipy-stack-osx-testing is only Mac.  There are pre-built wheels for Ubuntu and travis-ci - we use them for travis testing, as does scikit-image, thanks to some fruitful work by @blink1073:

* https://github.com/nipy/dipy/blob/master/.travis.yml#L36
* https://github.com/scikit-image/scikit-image/blob/master/tools/travis_before_install.sh

We use the following repo to build whatever travis ubuntu wheels we need:

* https://github.com/matthew-brett/travis-wheel-builder

The repo automatically uploads the wheels to http://travis-wheels.scikit-image.org/

It would be very easy / trivial to build current dev versions for numpy scipy, a little less trivial to automate that to happen every night or commit (for scipy / numpy), although python-Pillow recently set up build chaining to run the OSX tests after the linux tests: 

* https://github.com/python-pillow/pillow-wheels/pull/9

**Comment by josef-pkt** [comment]
Pauli announced the release ofscipy 0.15.1 which fixes the calc_lwork import issue

**Comment by josef-pkt** [comment]
fixed in scipy 0.15.1 and was already fixed in statsmodels master, closing
